taprpc/universerpc/universe.yaml

```diff
@@ -86,4 +86,8 @@ http:
 
     - selector: universerpc.Universe.UpdateSupplyCommit
       post: "/v1/taproot-assets/universe/update-supply-commit"
+      body: "*"
```

Comment:
To make these three endpoints a bit more RESTy, should we do this instead?

```
    - selector: universerpc.Universe.IgnoreAssetOutPoint
      post: "/v1/taproot-assets/universe/supply/ignore"
      body: "*"

    - selector: universerpc.Universe.UpdateSupplyCommit
      post: "/v1/taproot-assets/universe/supply/{group_key}"
      body: "*"

    - selector: universerpc.Universe.FetchSupplyCommit
      get: "/v1/taproot-assets/universe/supply/{group_key}"
```

Not sure if using the `{group_key}` placeholder that is of type `bytes` will work, but happy to test that if you agree to change the endpoints to the above.